pyproject.toml

```diff
@@ -88,6 +88,7 @@ PyYAML = "~5.4.1"
 social-auth-app-django = "^4.0.0"
 # Rendering of SVG images (for rack elevations, etc.)
 svgwrite = "~1.4.1"
+django-jinja = "^2.8.0"
```

Comment:
nit: please move this up so it fits in alphabetized order of dependencies, and add a brief comment. Thanks!